src/Api/Controllers/CiphersController.cs

```diff
@@ -143,7 +143,7 @@ public async Task<CipherMiniResponseModel> PostAdmin([FromBody]CipherCreateReque
             }
 
             var userId = _userService.GetProperUserId(User).Value;
-            await _cipherService.SaveAsync(cipher, userId, model.CollectionIds, true, false);
+            await _cipherService.SaveAsync(cipher, userId, model.Cipher.LastKnownRevisionDate, model.CollectionIds, true, false);
```

Comment:
Couldn't you use the `.ToOrganizationCipher()` extension method to populate `RevisionDate` and check that vs. using it as a separate parameter or would that break the `null` reverse compatibility?

Reply:
That was my original plan, but @kspearrin was pretty adamant that a division of responsibilities was important here.

The idea being that the Cipher is intended to be saved, whereas the parameter is intended for validation. We kind of have a similar example in ShareAsync, which already has some of this kind of validation on it.